### PR TITLE
docs(framework): add NestJS guide and icons

### PR DESCRIPTION
**Pull request by rifont**
### What?
* Add NestJS Framework handler documentations
* Add icons for all Novu Framework quickstart guides in the overview page

Closes https://github.com/novuhq/novu/issues/6552

### Screenshots
_Preview deployment_
https://novu-nv-4304-docs-feedback-add-informations-about-how-to-us.mintlify.app/quickstart/nestjs

### CODE REVIEW

File: mint.json [hunk 1]
```diff
@@ -81,9 +81,10 @@
       "pages": [
         "quickstart/overview",
         "quickstart/nextjs",
-        "quickstart/svelte",
-        "quickstart/remix",
         "quickstart/express",
+        "quickstart/remix",
+        "quickstart/nestjs",
+        "quickstart/svelte",
```
Comment (rifont) [inline mint.json:87]: Reordering here based on perceived frequency of usage, from most to least used.

File: snippets/quickstart/workflow.mdx [hunk 2]
```diff
@@ -14,9 +14,9 @@ export const testWorkflow = workflow('test-workflow', async ({ step, payload })
...
         }),
     });
-    }, {
-        payloadSchema: z.object({
-            userName: z.string().default('John Doe'),
-        }),
+}, {
+    payloadSchema: z.object({
+        userName: z.string().default('John Doe'),
+    }),
```
Comment (rifont) [inline snippets/quickstart/workflow.mdx:20]: Fixing some whitespace issues.

File: quickstart/remix.mdx [hunk 3]
```diff
@@ -9,6 +9,7 @@ import TestStep from "/snippets/quickstart/test.mdx";
 import PackagesStep from "/snippets/quickstart/packages.mdx";
 import SecretStep from "/snippets/quickstart/secret.mdx";
 import WorkflowCode from "/snippets/quickstart/workflow.mdx";
+import NextStepsStep from "/snippets/quickstart/next-steps.mdx";
```
Comment (rifont) [inline quickstart/remix.mdx:12]: This import was missing and previously resulted in the "Next steps" section not being rendered at all.

**Review by scopsy — APPROVED** [review]
Love it!

**Review by SokratisVidros — APPROVED** [review]

File: quickstart/nestjs.mdx [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+        The following example does not support NestJS dependency injection. If you need to `@Injectable` dependencies in your workflow definition, see [Advanced Usage](#advanced-usage-dependency-injection).
+
+        <CodeGroup>
+            ```typescript src/app.module.ts
+            import { Module } from '@nestjs/common';
+            import { NovuModule } from '@novu/framework/nest';
+            import { testWorkflow } from './novu/workflows';
+
+            @Module({
```
Comment (SokratisVidros) [inline quickstart/nestjs.mdx:30]: Please fix identation, we use 2 spaces.

File: snippets/quickstart/workflow.mdx [hunk 5]
```diff
@@ -14,9 +14,9 @@ export const testWorkflow = workflow('test-workflow', async ({ step, payload })
             subject: z.string().default('A Successful Test on Novu from {{userName}}'),
         }),
     });
-    }, {
-        payloadSchema: z.object({
-            userName: z.string().default('John Doe'),
-        }),
+}, {
+    payloadSchema: z.object({
+        userName: z.string().default('John Doe'),
```
Comment (SokratisVidros) [inline snippets/quickstart/workflow.mdx:19]: Indentation

**Pull request merged**